docs/releases/1.8.3.txt

```diff
@@ -71,3 +71,6 @@ Bugfixes
   using UUID primary keys (:ticket:`24912`).
 
 * Fixed removing ``unique_together`` constraints on MySQL (:ticket:`24972`).
+
+* Fixed crash when registering content type for valid Windows Bitmap image
```

Comment:
more understandable to users: Fixed crash when uploading bitmap images in forms.ImageField